js/angular-crmMailing/directives.js

```diff
@@ -28,6 +28,7 @@
           });
           scope.crmMailingConst = CRM.crmMailing;
           scope.ts = CRM.ts(null);
+          scope.helpFile = 'CRM/Mailing/Form/Upload';
```

Comment:
The "New Mailing" forms will get deleted, but there are a few other forms that will be kept (e.g. administration and subscription forms).